python_src/src/lamp_py/postgres/seed_metadata.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import logging
+import sys
+from typing import List
+
+import sqlalchemy as sa
+
+from lamp_py.runtime_utils.import_env import load_environment
+
+from .postgres_schema import (
+    MetadataLog,
+    SqlBase,
+    VehicleEventMetrics,
+    VehicleEvents,
+    VehicleTrips,
+)
+from .postgres_utils import DatabaseManager
+
+logging.getLogger().setLevel("INFO")
+
+DESCRIPTION = """Interact with Performance Manager RDS for Testing"""
+
+
+def parse_args(args: List[str]) -> argparse.Namespace:
+    """parse args for running this entrypoint script"""
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+
+    parser.add_argument(
+        "--env-file",
+        dest="env_file",
+        help="environment file containing rds connection parameters",
+        required=True,
+    )
```

Comment:
I don't imagine I'll be running this often from the command line, docker hostname resolution is a litter different between platforms and my preference would be to have this run automatically in the container with docker compose. With that in mind, the `--env-file` flag would have to not be required.